authorization-service/src/utils/policy.ts

```diff
@@ -0,0 +1,27 @@
+export const generatePolicy = (principalId: string, resource: string, effect = 'Allow') => ({
+    principalId,
+    policyDocument: {
+      Version: '2012-10-17',
+      Statement: [
+        {
+          Action: 'execute-api:Invoke',
+          Effect: effect,
+          Resource: resource
+        }
+      ]
+    }
+  });
+
+export const getCredentialsFromToken = (authorizationToken: string) => {
+  const [, encodedCreds = ''] = authorizationToken.split(' ');
+  const buff = Buffer.from(encodedCreds, 'base64');
+  const [username = '', password = ''] = buff.toString('utf-8').split(':');
```

Comment:
Wrong token format!
Expected:
unfalse=TEST_PASSWORD
Actual:
unfalse:TEST_PASSWORD

Reply:
I disagree. Please read carefully the task:
_This lambda should have at least one environment variable with credentials: {yours_github_account_login}=TEST_PASSWORD_
and
_Basic Authorization header: Authorization: Basic authorization_token where authorization_token is equal base64-encoded({yours_github_account_login}:TEST_PASSWORD)_

See the difference?